### PR TITLE
Issue12 add hotkey functions to brush tools

### LINKED ISSUE #12: Add Ctrl-C and Ctrl-V hotkey functions to Brush Tools
Add Ctrl-C and Ctrl-V hotkey functions to `BrushTool `and `FreehandScissorsTool`

### PR DESCRIPTION
**Pull request by VinceAdamo**
This PR sets up hotkeys for the `BrushTool ` and the `FreehandScissorsTool`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@altis/cornerstone-tools",
-  "version": "1.1.5",
+  "version": "1.1.6",
```
Comment (mkramers) [inline package.json]: This should be `1.1.7` I believe

File: src/tools/segmentation/BrushTool.js [hunk 2]
```diff
@@ -63,4 +73,22 @@ export default class BrushTool extends BaseBrushTool {
 
     external.cornerstone.updateImage(evt.detail.element);
   }
+
+  handleKeyboardCallback(evt) {
+    const { key, ctrlKey } = evt.detail;
```
Comment (mkramers) [inline src/tools/segmentation/BrushTool.js]: Given that we will likely add new hotkeys in the future, can you extract this into a re-usable function. Let's discuss the best approach to best do this.

**Review by mkramers — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (VinceAdamo) [reply package.json]: c3cc50154230ac21ba11c9f00428ad2d0a1b4897

File: src/tools/segmentation/BrushTool.js (hunk 2 above)
Comment (VinceAdamo) [reply src/tools/segmentation/BrushTool.js]: c3cc50154230ac21ba11c9f00428ad2d0a1b4897

**Review by mkramers — APPROVED** [review]

**Pull request merged**